oidc_auth/authentication.py

```diff
@@ -101,71 +139,77 @@ def get_jwt_value(self, request):
         auth = get_authorization_header(request).split()
         auth_header_prefix = api_settings.JWT_AUTH_HEADER_PREFIX.lower()
 
-        if not auth or smart_text(auth[0].lower()) != auth_header_prefix:
+        if not auth or smart_str(auth[0].lower()) != auth_header_prefix:
             return None
 
         if len(auth) == 1:
             msg = _('Invalid Authorization header. No credentials provided')
             raise AuthenticationFailed(msg)
         elif len(auth) > 2:
-            msg = _('Invalid Authorization header. Credentials string should not contain spaces.')
+            msg = _(
+                'Invalid Authorization header. Credentials string should not contain spaces.')
             raise AuthenticationFailed(msg)
 
         return auth[1]
 
+    # @cache(ttl=api_settings.OIDC_JWKS_EXPIRATION_TIME)
```

Comment:
Assume you meant to leave this uncommented, like the original `jwks_data`?

Reply:
@kerrermanisNL I am now able again to cash the output